app/models/tag.rb

```diff
@@ -201,4 +201,11 @@ def self.find_research_notes(tagnames, limit = 10)
         .order('node_revisions.timestamp DESC')
         .limit(limit)
   end
+
+  def followers_who_dont_follow_tags(tags)
+    tag_followers = self.subscriptions.map {|user| user.user_id}
```

Comment:
Why not just use `self.followers`? https://github.com/publiclab/plots2/blob/master/app/models/tag.rb#L135 -- maybe there's a good reason that I'm missing?